PollBuddy-Server/backend/routes/polls.js

```diff
@@ -1,64 +1,80 @@
-var createError = require("http-errors");
 var express = require("express");
 var router = express.Router();
 var mongoConnection = require("../modules/mongoConnection.js");
+const Joi = require("joi");
+const {createResponse, validateID} = require("../modules/utils");
```

Comment:
An interesting way to include a file that I don't think we've really used yet. Request for comments on this. It's the difference between `createResponse(x,y,z)` and `utils.createResponse(x,y,z)`. `utils.` follows other formatting more consistently, whereas without is a bit easier to write, and allows selective importing of only relevant functions (although that's debatably important in itself). I'm fine with either, just thought I should mention it since stylistic things are a topic recently.